collect_app/src/main/java/org/odk/collect/android/tasks/InstanceSyncTask.java

```diff
@@ -109,6 +111,10 @@ protected String doInBackground(Void... params) {
                     }
                 }
 
+                boolean instanceSyncFlag = PreferenceManager.getDefaultSharedPreferences(
+                        Collect.getInstance().getApplicationContext()).getBoolean(
```

Comment:
If you make `instanceSyncFlag` final it reduces the cognitive load of readers who might wonder if it changes before it is used far below.